docs/mesos/README.md

```diff
@@ -0,0 +1,27 @@
+# Mesos with Calico Networking
+Calico provides networking for your Mesos cluster. These guides will get you started.
+
+## Preparing Hosts
+[This guide](PrepareHosts.md) will walk you through preparing each host in your cluster to be compatible with Calico and Mesos.
+
+## Preparing External Services
+Calico utilizes etcd as its datastore, while Mesos utilizes ZooKeeper. [This guide](PrepareExternalServices.md) will walk you through setting up both services using Docker.
+
+## Mesos + Netmodules
+There are several ways to install Mesos with Net-Modules. 
```

Comment:
Lets add a quick one liner as to why we're doing this:
```
Calico works in conjunction with [Netmodules][net-modules], a Mesos Networking Module. The following guides will install Mesos with the necessary Netmodules libraries:
```

Also use the link structure that spike did to get that link to work as is: https://github.com/projectcalico/calico-mesos/blame/master/docs/deploying.md#L208